sample/src/main/java/com/polidea/rxandroidble2/sample/example2_connection/ConnectionExampleActivity.java

```diff
@@ -40,15 +36,15 @@
     SwitchCompat autoConnectToggleSwitch;
     private RxBleDevice bleDevice;
     private Disposable connectionDisposable;
-    private final CompositeDisposable compositeDisposable = new CompositeDisposable();
+    private Disposable mtuDisposable;
+    private Disposable connectionStateDisposable;
 
     @OnClick(R.id.connect_toggle)
     public void onConnectToggleClick() {
         if (isConnected()) {
             triggerDisconnect();
         } else {
             connectionDisposable = bleDevice.establishConnection(autoConnectToggleSwitch.isChecked())
```

Comment:
This one does not seem to be properly disposed

Reply:
This one does get disposed by calling `triggerDisconnect()` in `onPause()`.